packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -55,7 +55,9 @@ describe("JsonCursor", () => {
         });
 
         it("array", () => {
-            // TODO: should empty arrays report this key?
```

Comment:
I don't think we ever reached a decision on this. We probably should.

experimental shared-tree explicitly omitted all empty fields, which makes sense in a schemeless system, but we might want to make a different choice. That said, omitting empty fields can make enumerating children on nodes with many mostly empty fields faster, as well as make global, local and extra fields work in a more uniform way, and you can alwayse enumerate the schema instead if you want them included.

Reply:
Sorry, I mis-read your comment.  I thought you were asking if 'EmptyKey' should be reported.  I agree your TODO is still valid.

Reply:
@CraigMacomber - I generalized the TODO a bit, since I think it applies to any field of kind optional/sequence.